src/controller/tests/data_model/TestRead.cpp

```diff
@@ -109,10 +109,13 @@ CHIP_ERROR ReadSingleClusterData(const Access::SubjectDescriptor & aSubjectDescr
             // Use an incorrect attribute id for some of the responses.
             path.mAttributeId =
                 static_cast<AttributeId>(path.mAttributeId + (i / 2) + (responseDirective == kSendManyDataResponsesWrongPath));
-            AttributeValueEncoder::AttributeEncodeState state =
-                (apEncoderState == nullptr ? AttributeValueEncoder::AttributeEncodeState() : *apEncoderState);
-            AttributeValueEncoder valueEncoder(aAttributeReports, aSubjectDescriptor.fabricIndex, path,
-                                               kDataVersion /* data version */, aIsFabricFiltered, state);
+            AttributeEncodeState state;
+            if (apEncoderState != nullptr)
+            {
+                state = *apEncoderState;
+            }
+            AttributeValueEncoder valueEncoder(aAttributeReports, aSubjectDescriptor, path, kDataVersion /* data version */,
```

Comment:
Pre-existing, but I don't think this comment adds much here.

Same in a few places below.

Reply:
Removed some of the useless comments.